tags/all/families.csv

```diff
@@ -2328,11 +2289,6 @@ Edu VIC WA NT Beginner,/Quality/Drawing,100
 Edu VIC WA NT Beginner,/Quality/Spacing,100
 Edu VIC WA NT Beginner,/Script/Handwritten,100
 Edu VIC WA NT Beginner,/Script/Informal,60
-Ek Mukta,/Expressive/Business,89

```

Comment:
We serve Mukta instead. Guess this family didn't progress because we got the foundry to drop the "Ek" part since it's their foundry name